_posts/2017-04-27-Rust-1.17.md

```diff
@@ -0,0 +1,352 @@
+   3:     0x55c39a235324 - std::panicking::default_hook::h1670459d2f3f8843
+                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:367
+   4:     0x55c39a235afb - std::panicking::rust_panic_with_hook::hcf0ddb069e7beee7
+                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:555
+   5:     0x55c39a22e866 - std::panicking::begin_panic::heb433e9aa28a7408
+   6:     0x55c39a22e9bf - foo::main::hd216d4a160fcce19
+   7:     0x55c39a23d44a - __rust_maybe_catch_panic
+                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libpanic_unwind/lib.rs:98
+   8:     0x55c39a236006 - std::rt::lang_start::hd7c880a37a646e81
+                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panicking.rs:436
+                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/panic.rs:361
+                        at /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libstd/rt.rs:57
+   9:     0x55c39a22e9e9 - main
+  10:     0x7f5e5ed3382f - __libc_start_main
+  11:     0x55c39a22e6b8 - _start
+  12:                0x0 - <unknown>
+```
+
+is now instead
+
+```text
+thread 'main' panicked at 'explicit panic', foo.rs:2
+stack backtrace:
+   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
+             at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
+   1: std::sys_common::backtrace::_print
+             at /checkout/src/libstd/sys_common/backtrace.rs:71
+   2: std::panicking::default_hook::{{closure}}
+             at /checkout/src/libstd/sys_common/backtrace.rs:60
+             at /checkout/src/libstd/panicking.rs:355
+   3: std::panicking::default_hook
+             at /checkout/src/libstd/panicking.rs:371
+   4: std::panicking::rust_panic_with_hook
+             at /checkout/src/libstd/panicking.rs:549
+   5: std::panicking::begin_panic
+   6: foo::main
+   7: __rust_maybe_catch_panic
+             at /checkout/src/libpanic_unwind/lib.rs:98
+   8: std::rt::lang_start
+             at /checkout/src/libstd/panicking.rs:433
+             at /checkout/src/libstd/panic.rs:361
+             at /checkout/src/libstd/rt.rs:57
+   9: main
+  10: __libc_start_main
+  11: _start
+```
+
+By default. As the message says, you can set an environment variable to get the full
+backtrace. We may be able to do more cleanup in the future; see [this bug](https://github.com/rust-lang/rust/pull/40264) for more.
+
+See the [detailed release notes][notes] for more.
+
+#### Cargo features
+
+Other than the previously mentioned `build.rs` changes, Cargo has a few new improvements.
+[`cargo check --all`](https://github.com/rust-lang/cargo/pull/3731) and
+[`cargo run --package`](https://github.com/rust-lang/cargo/pull/3691) are two missing
+flags that are now supported.
```

Comment:
there's no specific new docs, as these do what `cargo check` and `cargo run` do as normal, but were just missing these standard flags